packages/edit-post/src/components/options-modal/options.js

```diff
@@ -51,7 +52,8 @@ export const EnablePublishSidebarOption = compose(
 		return {
 			onChange: ( isEnabled ) => ( isEnabled ? enablePublishSidebar() : disablePublishSidebar() ),
 		};
-	} )
+	} ),
+	ifViewportMatches( 'medium' ),
```

Comment:
It would be helpful to have a little comment here that explains that small viewports **always** have the publish sidebar.

Reply:
c567dee